python/nav/statemon/checker/HttpChecker.py

```diff
@@ -58,12 +58,13 @@ def connect(self, ip, port):
 
     def execute(self):
         ip, port = self.get_address()
-        url = self.args.get('url', '')
+        url = self.args.get('url', '/')
```

Comment:
I've started using `dict.get(foo) or fallback` instead because some dicts always return None if key not found and I have no idea why.